src/modules/accounting/accounting.py

```diff
@@ -108,13 +110,13 @@ def build_report(self, blockstamp: ReferenceBlockStamp) -> tuple:
 
     def is_main_data_submitted(self, blockstamp: BlockStamp) -> bool:
         # Consensus module: if contract got report data (second phase)
-        processing_state = self._get_processing_state(blockstamp)
+        processing_state = self.report_contract.get_processing_state(blockstamp.block_hash)
         logger.debug({'msg': 'Check if main data was submitted.', 'value': processing_state.main_data_submitted})
         return processing_state.main_data_submitted
 
     def can_submit_extra_data(self, blockstamp: BlockStamp) -> bool:
```

Comment:
A bit inconsistent with the other names: `is_main_data_submitted` and `is_contract_reportable` but `can_submit_extra_data`

Reply:
`is_extra_data_submittable` looks weird =)
and we can use `is_extra_data_submitted`, because it might be not delivered because main data is not submitted